cmd/create/decision/cmd_test.go

```diff
@@ -0,0 +1,97 @@
+package decision
+
+import (
+	"fmt"
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "github.com/openshift-online/ocm-sdk-go/accesstransparency/v1"
+	"github.com/openshift-online/ocm-sdk-go/testing"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/output"
+	. "github.com/openshift/rosa/pkg/test"
+)
+
+var _ = Describe("rosa attach policy", func() {
+	Context("Create Command", func() {
+		It("Returns Command", func() {
+
+			cmd := NewCreateDecisionCommand()
+			Expect(cmd).NotTo(BeNil())
+
+			Expect(cmd.Use).To(Equal(use))
+			Expect(cmd.Example).To(Equal(example))
+			Expect(cmd.Short).To(Equal(short))
+			Expect(cmd.Long).To(Equal(long))
+			Expect(cmd.Args).NotTo(BeNil())
+			Expect(cmd.Run).NotTo(BeNil())
+
+			flag := cmd.Flags().Lookup("access-request")
+			Expect(flag).NotTo(BeNil())
+			flag = cmd.Flags().Lookup("decision")
+			Expect(flag).NotTo(BeNil())
+			flag = cmd.Flags().Lookup("justification")
+			Expect(flag).NotTo(BeNil())
+		})
+	})
+
+	Context("Execute command", func() {
+		var (
+			t       *TestingRuntime
+			options *DecisionOptions
+		)
+
+		BeforeEach(func() {
+			t = NewTestRuntime()
+			options = &DecisionOptions{
+				accessRequest: "fake-id",
+			}
+			output.SetOutput("")
+			interactive.SetEnabled(false)
+		})
+
+		AfterEach(func() {
+			output.SetOutput("")
+			interactive.SetEnabled(false)
```

Comment:
I think we could get away with setting `interactive.SetEnabled(false)` only in the BeforeEach, but if im wrong no big deal

Reply:
I removed all interactive related code since interactive mode is not supported for this command